src/cart/crt_corpc.c

```diff
@@ -42,14 +42,15 @@ crt_corpc_info_init(struct crt_rpc_priv *rpc_priv,
 	D_ASSERT(rpc_priv != NULL);
 	D_ASSERT(grp_priv != NULL);
 
-	D_ALLOC_PTR(co_info);
+	co_info = d_mm_alloc(sizeof(*co_info));
```

Comment:
should add a d_alloc like macro combo?  We have log parsing leak detection

Reply:
Not sure we need more macros... Personally I don't like macros. I should always remember what exactly represent this macro. But potentially I can add one.

Reply:
For leak detection we should track d_mm_fini() because of whole list free there.

Reply:
I agree with Jeff, macros here would be better for a number of reasons.

- The _PTR() suffix automatically takes the size, improving readability and reducing the chance for mistakes
- It improves logging, so we can tell the flow through the code from reading the logs.
- It would allow us to use the fault injection framework consistently.